ci/pipelines/deployer-updater.yaml

```diff
@@ -0,0 +1,116 @@
+resource_types:
+- name: github
+  type: registry-image
+  source:
+    repository: "govsvc/concourse-github-resource"
+    tag: "0.0.1551114195"
+
+resources:
+- name: platform
+  type: github
+  source:
+    uri: https://github.com/alphagov/gsp-terraform-ignition.git
+    paths:
+      - pipelines/**/*.yaml
```

Comment:
can we be specific here? ... we know we only need `pipelines/deployer/deployer.yaml` right?

Reply:
Now :trollface:

Sure, I'll change that.